.github/pull_request_template.md

```diff
@@ -1,8 +1,13 @@
 Thank you for helping make the Dapr documentation better!
 
-If you are a new contributor, please see the [this contribution guidance](https://github.com/dapr/docs/blob/master/contributing/README.md) which helps keep the Dapr documentation readable, consistent and useful for Dapr users.
+**Please follow this checklist before submitting:**
 
-If you are contributing a new authored "How To" article, [this template](https://github.com/dapr/docs/blob/master/contributing/howto-template.md) was created to assist you.
+- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
```

Comment:
shouldn't this be a shortcode? :)

Reply:
Since this template is rendered in GitHub and not built by hugo we have to link via URL